src/parsers/trades/gate.io/index.ts

```diff
@@ -0,0 +1,93 @@
+import { getCSVData } from '../../';
+import { EXCHANGES, IImport, IPartialTrade, ITrade } from '../../../types';
+import { createDateAsUTC, createID } from '../../utils';
+
+interface IGateIO {
+    No: string;
+    'Account Type': string;
+    Time: string;
+    'Action type': string;
+    Currency: string;
+    'Order id': string;
+    'Change amount': string;
+    Amount: string;
+    'Additional Info': string;
+}
+
+export default async function processData(importDetails: IImport): Promise<ITrade[]> {
+    const data: IGateIO[] = await getCSVData(importDetails.data) as IGateIO[];
+    const internalFormat: ITrade[] = [];
+    if (data.length < 1) {
+        return internalFormat;
+    }
+    let feeTrade = data[0];
+    let filledTrade = data[0];
+    let pointTrade = data[0];
+    let lineContinuity = 0;
+    for (const trade of data) {
+        const tradeToAdd: IPartialTrade = {};
+        tradeToAdd.date = createDateAsUTC(new Date(trade.Time)).getTime();
```

Comment:
Id move these all into the partial trade setting instead:

```
const tradeToAdd: IPartialTrade = {
  date: createDateAsUTC(new Date(trade.Time)).getTime(),
  exchange: EXCHANGES.GateIO,
};
```

Reply:
Sure I can do that